docs/website/pages/index.vue

```diff
@@ -1,57 +1,55 @@
 <template>
   <div id="landing-page" class="c-rich-text">
-    <div class="w-1/2 h-auto mx-auto mt-10 mb-20">
-      <div class="flex flex-col justify-center">
-        <div class="text-center p-0 m-0">
-          <h1>What is Talos?</h1>
+    <div class="w-auto md:w-1/2 h-auto mx-auto mt-10 mb-20 px-4">
+      <div class="text-center p-0 m-0">
+        <h1>What is Talos?</h1>
+      </div>
+      <div class="flex flex-wrap justify-center items-center">
```

Comment:
nit: IMHO, the text looks funky without `text-center`. At least in this hero section it does. The subsequent removal of `text-center` looks good.

Reply:
I think it might be ugly because of the "river". 
<img width="794" alt="Screen Shot 2019-10-24 at 6 04 16 PM" src="https://user-images.githubusercontent.com/423815/67529004-eb966d00-f688-11e9-98ca-267329175f4a.png">

What about breaking it up into two paragraphs, like this?

<img width="790" alt="Screen Shot 2019-10-24 at 6 04 56 PM" src="https://user-images.githubusercontent.com/423815/67529040-00730080-f689-11e9-8b8c-569d4be748ec.png">

I find the rows of centered text a little bit difficult to read.

Reply:
LGTM 👍